docs/api/session.md

```diff
@@ -181,15 +181,13 @@ Clears the data of web storages.
 
 #### `ses.setProxy(config, callback)`
 
-* `config` String
+* `config` Object
+  * `pacScript` String - The URL associated with the PAC file.
+  * `proxyRules` String - Rules indicating which proxies to use.
```

Comment:
`automatic` and `manual` are not appropriate for option names, they are only meaningful when used as values. And when writing code, I would prefer:

```js
setProxy({pacScript: 'http://some.host'});
```

to 

```js
setProxy({automatic: 'http://some.host'});
```

The latter just makes no sense.